src/methods.ts

```diff
@@ -91,6 +91,9 @@ export interface MessageAttachment {
     type: string;
     text?: string;
   }[];
+  attachment_type?: 'default';
```

Comment:
while i see that this is an accepted field, there are no values other than 'default', so i don't think it adds any value to type it (i'm not sure there will ever be any other possible values, and each additional property has a visual cost when your editor is trying to give you hints)

Reply:
thanks, I understand, but our UX team uses [Walkiebot](https://walkiebot.co/) for designing flows and messages. We also plan to fetch those messages designed by Walkiebot automatically to our integrations (using the Walkiebot API), but there is a problem - Walkiebot generates JSON/JS of designed attachments with `attachmet_type` field included (see the attached screenshot), therefore the output is not compatible with this SDK and we have to manually remove this field from every exported message. I can contact Walkiebot and ask them to remove this field, but it seems, they strictly stick to Slack API specs, so maybe as a first step is good to remove this unnecessary field from Slack API documentation.
![walkiebot](https://user-images.githubusercontent.com/1623257/38722814-814a00a2-3eff-11e8-8dfd-ca0333189f55.PNG)

Reply:
thanks, we use TypeScript for all our projects - but I thought before that our attachments were incompatible with the `MessageAttachment` because of missing the `attachment_type` field, but as I see now, my assumption is not true (thanks to the `AuxiliaryArguments`). Our incompatibility comes from missing types in the nested field `MessageAttachment.actions` - the current type definition for `actions` contains only two fields:
```
actions?: {
    type: string;
    text?: string;
  }[];
```
but the attachment actions in our project contains additional fields. So I will prepare a separate PR also with complete definitions for the attachment actions.